File: backend/src/main/java/com/ll/hotel/global/jwt/dto/JwtProperties.java
package com.ll.hotel.global.jwt.dto;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Data
@Component
@ConfigurationProperties(prefix = "custom.jwt")
public class JwtProperties {
    private String secret;
    private long accessTokenExpiration;
    private long refreshTokenExpiration;
}

File: backend/src/main/java/com/ll/hotel/global/jwt/dto/JwtProperties.kt
package com.ll.hotel.global.jwt.dto

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "custom.jwt")
data class JwtProperties(
    var secret: String = "",
    var accessTokenExpiration: Long = 0,
    var refreshTokenExpiration: Long = 0
) 